src/api.ts

```diff
@@ -142,6 +142,12 @@ export default class Chromeless<T extends any> implements Promise<T> {
     return this
   }
 
+  setDocumentContent(html: string): Chromeless<T> {
```

Comment:
I definitely see your point. I'm open to other suggestions for names for this method. Perhaps `setHTML()`?